ontology/yaml/resources/HVAC/entity_types/HWS.yaml

```diff
@@ -266,4 +266,11 @@ HWS_SWTC_CPC:
   uses:
   - circulation_pump_speed_percentage_command
   - circulation_pump_speed_percentage_setpoint
-  - failed_alarm
\ No newline at end of file
+  - failed_alarm
+  
+HWS_ST:
+  description: "Simple hot water system with status only"
+  implements:
+  - HWS
+  uses:
+  - water_heating_status
```

Comment:
`run_command`?

Reply:
in our device points list, there is no run_command sir, just the status if heating or not (no control).